### PR TITLE
Minimum Password Age Function

### PR DESCRIPTION
**Pull request by Kirat31**

### CODE REVIEW

File: app/Http/Controllers/Auth/ChangePasswordController.php [hunk 1]
```diff
@@ -24,8 +24,12 @@ public function edit()
 
     public function update(UpdatePasswordRequest $request)
     {
-	$info = $request->validated();
-	$info['password_changed_at'] = Carbon::now();
+        $info = $request->validated();
+        $info['updated_at'] = Carbon::parse();
+	    $info['password_changed_at'] = Carbon::now();
```
Comment (ashikali) [inline app/Http/Controllers/Auth/ChangePasswordController.php]: password_changed_at must be taken from that user's table row not the current date

File: app/Http/Controllers/Auth/ChangePasswordController.php [hunk 2]
```diff
@@ -5,6 +5,7 @@
 use App\Http\Controllers\Controller;
 use App\Http\Requests\UpdatePasswordRequest;
 use App\Http\Requests\UpdateProfileRequest;
+use App\Model\User;
```
Comment (ashikali) [inline app/Http/Controllers/Auth/ChangePasswordController.php]: Do we need to include the App\Model\User. Since you are calling auth() not User

Comment (Kirat31) [reply app/Http/Controllers/Auth/ChangePasswordController.php]: No, we do not. My apologies. I thought of calling it through User first, but then switched it to auth().

File: app/Models/User.php [hunk 3]
```diff
@@ -9,6 +9,7 @@
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Model;
```
Comment (ashikali) [inline app/Models/User.php]: Do we need to include Model here ?

Comment (Kirat31) [reply app/Models/User.php]: Apologies for the mistake. You are right, we don't need to include the model here. I shall make the necessary changes.

**Pull request merged**